README.md

```diff
@@ -101,12 +101,31 @@ The image below shows the cloud vendor agnostic technical architecture for the H
 
 <a href="data-browser/README.md">Please view the README for HMDA Data Browser</a>
 
+## Installations
+Before running the HMDA Platform, make sure to have the following installed:
+1. Homebrew - https://brew.sh/
+2. Docker - ```bash brew install docker ```
+3. Docker Desktop - https://docs.docker.com/desktop/install/mac-install/
+4. Java (version 13.0.2) for MacOS - [https://www.java.com/en/download/](https://www.oracle.com/java/technologies/javase/jdk13-archive-downloads.html)
+5. Scala (version 2.12 for compatibility issues) - ```bash brew install scala@2.12 ```
+6. sdk - https://sdkman.io/install
+Next, use sdk to install sbt instead of brew (it won't work with brew):
+
+```bash
+sdk install sbt
+```
+Clone the repo and go into the repo directory:
+```bash
+git clone https://github.com/cfpb/hmda-platform.git
+cd hmda-platform
+```
+Before running with sbt, make sure your sbt version matches with the project version. Exit the project directory and run sbt -version to check the version. Go back to the project directory, go to project directorym then edit the build.properties to match your actual sbt version. NOTE: Please do NOT commit any changes to this file to the remote repo, revert it before pushing
```

Comment:
I don't think updating this is necessary, as long as your script version when you run `sbt -version` is higher than the version listed in `project/build.properties`

Reply:
That makes sense. Should we note that or remove this paragraph altogether?

Reply:
I think it would be good to note that we should check what version of sbt is in `project/build.properties` and make sure to install sbt at that version or higher.